karma.conf.js

```diff
@@ -94,7 +107,8 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['Firefox', 'Chrome'],
+//    browsers: ['Firefox', 'Chrome'],
+    browsers: ['Chrome'],
```

Comment:
was this intentional?

Reply:
Line 20 of the karma.conf.js file tells Karma to look for `'test/lib/**/*.js'`. But there is no test/lib directory.

Reply:
fixed in new [commit](https://github.com/opentok/opentok-rtc/pull/675/commits/be36d809175b8545f1fec2276f4b8857cae0dfb9)